eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -34,6 +35,8 @@
  *
  * @since 0.1
  */
+@SuppressWarnings({"PMD.AvoidDuplicateLiterals",
+    "PMD.ConstructorOnlyInitializesOrCallOtherConstructors"})
```

Comment:
@yasamprom I think you don't need these suppressions anymore, since now your code looks much better